new-era-commands/slash/notes.js

```diff
@@ -0,0 +1,17 @@
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('notes')
+    .setDescription('Things to consider about taking notes'),
+  execute: async (interaction) => {
+    const freelanceEmbed = new EmbedBuilder()
+      .setColor('#cc9543')
+      .setTitle('To note, or not to note, that is the question')
+      .setDescription(`
+Please read this [discord message about what we think you should consider about taking notes](https://discord.com/channels/505093832157691914/505093832157691916/768161823366578176)
```

Comment:
Nit: Discord should be capitalized.

Also this might also need to be updated in one of your other PRs which I already merged :grimacing: could you double check and fix that as well? I'm fine with that being done under this PR if you'd like.

Reply:
Great catch and fixed. I will look through the merged ones to see if I missed it there as well 🙈